git_machete/github.py

```diff
@@ -148,8 +150,22 @@ def create_pull_request(org: str, repo: str, head: str, base: str, title: str, d
         'body': description,
         'draft': draft
     }
-    pr = fire_github_api_get_request('POST', f'/repos/{org}/{repo}/pulls', token, request_body)
-    return parse_pr_json(pr)
+    prs: List[GitHubPullRequest] = derive_pull_requests(org, repo)
+    to_load: GitHubPullRequest = GitHubPullRequest(1, 'user', base, head, '')
+    if not is_pr_already_created(to_load, prs):
+        pr = fire_github_api_get_request('POST', f'/repos/{org}/{repo}/pulls', token, request_body)
+        return parse_pr_json(pr)
+    else:
+        raise MacheteException('Given Pull Request is already created!')
+
+
+def is_pr_already_created(pull: GitHubPullRequest, pull_requests: List[GitHubPullRequest]) -> bool:
```

Comment:
Same here, private (TBD if the prefix should be `_` or `__`, I'm not certain what is the proper convention for module-private functions, as opposed to private class methods)